cg/apps/scout/scoutapi.py

```diff
@@ -350,6 +350,13 @@ def upload_rna_genome_build(
         except CalledProcessError as error:
             raise ScoutUploadError("Something went wrong when updating rna genome build") from error
 
+    def upload_rna_delivery_report(self, case_id: str, report_path: str) -> None:
+        """Load a fusion report into a case in the database."""
+
+        self.upload_report(
+            case_id=case_id, report_path=report_path, report_type="rna_delivery_report"
```

Comment:
Maybe add "rna_delivery_report" as a constant in `ScoutCustomCaseReportTags`

Reply:
Done